examples/simple.html

```diff
@@ -6,7 +6,7 @@
       a.async=1;
       a.src=g;
       m.parentNode.insertBefore(a,m)
-  })(window,document,'script','./public/bite.v1.js','ba');
+  })(window,document,'script','http://localhost:3000/bite.v1.js','ba');
```

Comment:
The path should be `http://local host:3000/v1/bite.js` now that it’s served by `now dev`